save-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.entities
```

Comment:
Please also add to readme in save-common, what we have discussed about common sources: entities can be placed in jvmMain if they are not intended to be used on frontend, and if usage of platform-specific APIs becomes to complex, we'll need to create a corresponding common class (which will be serizlizable) for the entity